frontend/tests/e2e/features/edit-fishbowl/edit-fishbowl.ts

```diff
@@ -99,6 +95,5 @@ Then('sees a placeholder with Enter Fishbowl button', () => {
 });
 
 Then('clicks on placeholders Enter Fishbowl link', () => {
-  cy.wait(2000);
```

Comment:
we should avoid this kind of waits (it is better to wait for a query or a mutation)